editor/project_manager.cpp

```diff
@@ -2093,7 +2093,7 @@ void ProjectManager::_run_project_confirm() {
 		const String &selected = selected_list[i].project_key;
 		String path = EditorSettings::get_singleton()->get("projects/" + selected);
 
-		if (!DirAccess::exists(path + "/.import")) {
+		if (!DirAccess::exists(path.plus_file(IMPORTED_FILES_PATH.right(6)))) {
```

Comment:
That's not really clear.

Reply:
I added a comment, should be clear now. Before I had a 2nd constant which ended in `_NORES` but I realized we could replace it in most cases except for this one, so I just hacked this together to chop off the leading `res://`.